composer.json

```diff
@@ -31,11 +31,11 @@
   "require-dev": {
     "phpunit/phpunit": "4.8.*",
     "squizlabs/php_codesniffer": "2.*",
-    "phpdocumentor/phpdocumentor": "2.*"
+    "phpdocumentor/phpdocumentor": "2.8.*"
   },
   "autoload": {
     "psr-4": {
-      "Google\\Cloud\\": "src"
+      "Gcloud\\": "src"
```

Comment:
We have been using the `Google\Cloud` for our samples and getting started docs. I prefer this namespace to `Gcloud`. It's a minor thing, but consistency is important.

Was there a specific reason for switching?